contracts/test/utils.ts

```diff
@@ -50,9 +55,42 @@ export async function setNextBlockTimestamp(timestamp: BigNumberish): Promise<nu
   return timestamp;
 }
 
+// --- Uniswap SDK Helpers ---
+export async function encodeRoute(tokens: SupportedToken[]): Promise<string> {
+  const route = await getRoute(tokens);
+  return encodeRouteToPath(route, false);
+}
+
+async function getRoute(tokens: SupportedToken[]): Promise<Route<Token, Token>> {
+  const poolPromises: Promise<Pool>[] = [];
+  tokens.forEach((_, index) => {
+    if (index > 0) poolPromises.push(getPoolInstance(tokens[index - 1], tokens[index]));
```

Comment:
would we want to make this cleaner for the sake of readability 

```suggestion
    const from_token = tokens[index];
    const to_token = tokens[index+1];
    poolPromises.push(
     getPoolInstance(from_token, to_token)
    );
```

Reply:
I know @apbendi would agree with you here 😛

So fixed in ed764701e4f064c93f2b6036e292895a3cf2f315

Reply:
yaaasss 🎉  😜